cmd/flags.go

```diff
@@ -182,6 +152,21 @@ func flagsProjectGenerate() []cli.Flag {
 			Usage:   "create SOPS age keys for generated project structure",
 			Aliases: []string{"c"},
 		},
+		&cli.StringSliceFlag{
```

Comment:
suggest using `list` instead of `list of` because these are all verbs.

Reply:
fixed